### PR TITLE
Update Node instructions for serverless with AWS Lambda

### PR DESCRIPTION
**Pull request by MimiDumpling**
Some Node customers need a serverless option with AWD Lambda. Created a page for them to use as a reference.

image

### CODE REVIEW

File: src/collections/_documentation/platforms/node/serverless.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+title: Serverless
+sidebar_order: 7
+---
+
+## AWS Lambda
+
+To set up Sentry error logging for a Lambda function, build a wrapper.
+
+```javascript
+    "use strict";
```
Comment (kamilogorek) [inline src/collections/_documentation/platforms/node/serverless.md]: Code looks good, although please pass it through the code style formatter. You can use this one and paste your code example directly - https://prettier.io/playground/

File: src/collections/_documentation/platforms/node/serverless.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    function sentryHandler(lambdaHandler) {
+    return async event => {
+        try {
+        return await lambdaHandler(event);
+        } catch (error) {
+        Sentry.captureException(error);
+        await Sentry.flush(2000);
+        return {
+            body: JSON.stringify({
+            err: error.message,
```
Comment (kamilogorek) [inline src/collections/_documentation/platforms/node/serverless.md]: According to the docs https://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-mode-exceptions.html it should be `${error. errorType}: ${error.errorMessage}`

File: src/collections/_documentation/platforms/node/serverless.md (hunk 1 above)
Comment (MimiDumpling) [reply src/collections/_documentation/platforms/node/serverless.md]: Ohhh nice. Thanks!

File: src/collections/_documentation/platforms/node/serverless.md (hunk 2 above)
Comment (MimiDumpling) [reply src/collections/_documentation/platforms/node/serverless.md]: Oh nice. How would you revise the code snippet?

Comment (kamilogorek) [reply src/collections/_documentation/platforms/node/serverless.md]: `return error;` :)

Comment (MimiDumpling) [reply src/collections/_documentation/platforms/node/serverless.md]: Lol. 😛 Updated.

**Comment by MimiDumpling** [comment]
@kamilogorek made the requested changes. Lemme know if there are other ways to improve the page. 😄 Thanks!

**Pull request merged**